src/components/Dashboard/TaskItem/utils.js

```diff
@@ -0,0 +1,23 @@
+export const getTaskRef = (props) => {
+  const { taskListRef, id } = props;
+  const taskRef = taskListRef.child(`${id}`);
+  return taskRef;
+};
+
+export const getTaskStyleByPriority = (props) => {
+  const { priority, status } = props;
+  let style = status === 'Done' ? 'taskItemDone' : 'taskItem';
```

Comment:
it's common to use camelCase for naming variables, but kebab-case for css classes

camelCase
snake_case
kebab-case

FYI: https://medium.freecodecamp.org/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849